public/components/app.tsx

```diff
@@ -30,14 +52,117 @@ export const App = ({
   chrome,
   plugins
 }: ObservabilityAppDeps) => {
+
+  // event explorer states
+  const initialTabId: string = getTabId(TAB_ID_TXT_PFX);
+  const [tabIds, setTabIds] = useState<Array<string>>([initialTabId]);
+  const [queries, setQueries] = useState<ITabQueries>({
+    [initialTabId]: {
+      [RAW_QUERY]: ''
+    }
+  });
+  const [queryResults, setQueryResults] = useState<ITabQueryResults>({
+    [initialTabId]: {}
+  });
+  const [fields, setFields] = useState<IExplorerTabFields>({
+    [initialTabId]: {
+      [SELECTED_FIELDS]: [],
+      [UNSELECTED_FIELDS]: []
+    }
+  });
+  const curQueriesRef = useRef(queries);
+  const [curSelectedTabId, setCurSelectedTab] = useState<string>(initialTabId);
+
+  function getTabId (prefix: string) {
+    return _.uniqueId(prefix);
+  }
+
+  const parentBreadcrumb = {
+    text: observabilityTitle,
+    href: 'observability#/'
+  };
+
   return (
-    <I18nProvider>
-      <>
-        <LogExplorer
-          http={ http }
-          plugins={ plugins }
-        />
-      </>
-    </I18nProvider>
+    <HashRouter>
+      <I18nProvider>
+        <>
+          <Switch>
+            <Route 
+              exact
+              path={['/', '/application_analytics', '/application_analytics/home']}
+              render={(props) => {
+                chrome.setBreadcrumbs([
+                  parentBreadcrumb,
+                  {
+                    text: 'Application analytics',
+                    href: 'observability#/application_analytics'
```

Comment:
do you need the `observability` before `#` in breadcrumb?

Reply:
Right, it should be removed, missed it.